src/maps/Map.tsx

```diff
@@ -42,6 +42,8 @@ export interface VectorMapProps {
   checkedLayers?: string[];
   /** Layer IDs to 'select' with the 'aria-current' attribute. */
   currentLayers?: string[];
+  /** Added to support react 18 */
+  children?: React.ReactNode;
 }
 
 const VectorMap: React.FC<VectorMapProps> = ({
```

Comment:
You could also make this a `React.FC<PropsWithChildren<VectorMapProps>>` instead of adding the children to the Props interface. But doesn't change much.

Reply:
Yes, but as you mention: I wouldn't change much...